### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Gelma**
Hi guys,
  thanks a lot for your work.

   Here a stupid patch about typos.

Thanks a lot,
Gelma

### CODE REVIEW

File: cmd/zpool/os/linux/zpool_vdev_os.c [hunk 1]
```diff
@@ -239,7 +239,7 @@ check_sector_size_database(char *path, int *sector_size)
 	io_hdr.dxferp = inq_buff;
 	io_hdr.cmdp = inq_cmd_blk;
 	io_hdr.sbp = sense_buffer;
-	io_hdr.timeout = 10;		/* 10 milliseconds is ample time */
+	io_hdr.timeout = 10;		/* 10 milliseconds is sample time */
```
Comment (freqlabs) [inline cmd/zpool/os/linux/zpool_vdev_os.c:242]: ample is correct.

File: module/os/linux/zfs/abd_os.c [hunk 2]
```diff
@@ -34,7 +34,7 @@
  *
  *  (1) They avoid use of kmem_*, preventing performance problems where running
  *      kmem_reap on very large memory systems never finishes and causes
- *      constant TLB shootdowns.
+ *      constant TLB shutdowns.
```
Comment (freqlabs) [inline module/os/linux/zfs/abd_os.c:37]: shootdowns is correct.

File: module/zfs/arc.c [hunk 3]
```diff
@@ -6361,7 +6361,7 @@ arc_release(arc_buf_t *buf, void *tag)
 			VERIFY(!arc_buf_is_shared(lastbuf));
 
 			/*
-			 * First, sever the block sharing relationship between
+			 * First, serve the block sharing relationship between
```
Comment (freqlabs) [inline module/zfs/arc.c:6364]: sever is correct.

File: module/zfs/dsl_dataset.c [hunk 4]
```diff
@@ -65,7 +65,7 @@
 
 /*
  * The SPA supports block sizes up to 16MB.  However, very large blocks
- * can have an impact on i/o latency (e.g. tying up a spinning disk for
+ * can have an impact on i/o latency (e.g. trying up a spinning disk for
```
Comment (freqlabs) [inline module/zfs/dsl_dataset.c:68]: tying is correct.

File: module/zfs/zio.c [hunk 5]
```diff
@@ -4659,7 +4659,7 @@ zio_done(zio_t *zio)
 		/*
 		 * "The Godfather" I/O monitors its children but is
 		 * not a true parent to them. It will track them through
-		 * the pipeline but severs its ties whenever they get into
+		 * the pipeline but serves its ties whenever they get into
```
Comment (freqlabs) [inline module/zfs/zio.c:4662]: severs is correct.

File: tests/zfs-tests/tests/functional/cli_root/zfs_get/zfs_get_005_neg.ksh [hunk 6]
```diff
@@ -62,7 +62,7 @@ typeset val_props=("${v_props[@]}" \
     "${userquota_props[@]}")
 set -f	# Force shell does not parse '?' and '*' as the wildcard
 typeset inval_opts=(P R h ? *)
-typeset inval_props=(Type 0 ? * -on --on readonl time USED RATIO MOUNTED)
+typeset inval_props=(Type 0 ? * -on --on readonly time USED RATIO MOUNTED)
```
Comment (freqlabs) [inline tests/zfs-tests/tests/functional/cli_root/zfs_get/zfs_get_005_neg.ksh:65]: readonl is intentionally incorrect.

File: tests/zfs-tests/tests/functional/cli_root/zfs_inherit/zfs_inherit_002_neg.ksh [hunk 7]
```diff
@@ -63,7 +63,7 @@ if is_freebsd; then
 else
 	props+=("zoned")
 fi
-set -A illprops "recordsiz" "mountpont" "sharen" "compres" "atme" "blah"
+set -A illprops "recordsiz" "mountpoint" "sharen" "compress" "atme" "blah"
```
Comment (freqlabs) [inline tests/zfs-tests/tests/functional/cli_root/zfs_inherit/zfs_inherit_002_neg.ksh:66]: "mountpont" and "compres" are intentionally incorrect.

File: cmd/zpool/os/linux/zpool_vdev_os.c (hunk 1 above)
Comment (Gelma) [reply cmd/zpool/os/linux/zpool_vdev_os.c]: Thanks a lot for the review! I'm going to upload reverts.

File: module/os/freebsd/zfs/zfs_znode.c [hunk 8]
```diff
@@ -1099,7 +1099,7 @@ zfs_rezget(znode_t *zp)
 	 * If the file has zero links, then it has been unlinked on the send
 	 * side and it must be in the received unlinked set.
 	 * We call zfs_znode_dmu_fini() now to prevent any accesses to the
-	 * stale data and to prevent automatical removal of the file in
+	 * stale data and to prevent automatically removal of the file in
```
Comment (ahrens) [inline module/os/freebsd/zfs/zfs_znode.c:1102]:
```suggestion
	 * stale data and to prevent automatic removal of the file in
```

Comment (Gelma) [reply module/os/freebsd/zfs/zfs_znode.c:1102]: Thanks a lot @ahrens!

**Comment by behlendorf** [comment]
@Gelma thanks!  Can you please just address the cstyle warning.

```
./module/os/freebsd/zfs/zfs_vnops.c: 388: line > 80 characters
```

**Comment by Gelma** [comment]
> @Gelma thanks! Can you please just address the cstyle warning.
> 
> ```
> ./module/os/freebsd/zfs/zfs_vnops.c: 388: line > 80 characters
> ```

Ok, I split on two lines. I'm going to send new commit.

**Comment by behlendorf** [comment]
@Gelma one last thing.  Can you please squash your changes in to a single commit, add your [signed-off-by](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by) and then update the PR.

**Comment by Gelma** [comment]
> @Gelma one last thing. Can you please squash your changes in to a single commit, add your [signed-off-by](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by) and then update the PR.

Thanks a lot. I'm going to upload new version.

**Pull request merged**